include/image.mk

```diff
@@ -566,6 +566,8 @@ Target-Profile-hasImageMetadata: $(if $(foreach image,$(IMAGES),$(findstring app
 Target-Profile-SupportedDevices: $(SUPPORTED_DEVICES)
 Target-Profile-Description:
 $(DEVICE_DESCRIPTION)
+$(if $(value DEVICE_FLASH),Flash: $(DEVICE_FLASH)mb)
+$(if $(value DEVICE_RAM),Ram: $(DEVICE_RAM)mb)
```

Comment:
Maybe capitalize RAM. I would strongly suggest to replace "mb" with " MiB", so
$(if $(value DEVICE_RAM),RAM: $(DEVICE_RAM) MiB)